PointerStar/Client/Pages/Room.razor

```diff
@@ -173,7 +144,27 @@
         await base.OnInitializedAsync();
         if (RoomId is { } roomId)
         {
-            User user = new(Guid.NewGuid(), Name ?? $"User {new Random().Next()}");
+            if(string.IsNullOrEmpty(Name))
+            {
+                Open();
+            }
+            else
+            {
+                await JoinRoom();
+            }
+        }
+    }
+
+    private async Task JoinRoom()
+    {
+        if(IsModalOpen)
+        {
+            Close();
+        }
+
+        User user = new(Guid.NewGuid(), Name ?? $"User {new Random().Next()}");
```

Comment:
Should the user's name be used here?

Reply:
@Keboo  it is being used. the `@bind="Name"` in line 172 mean anythign typed in the input box goes into the Name property. continue this on PR !51 If you need to